context/app/static/js/pages/Iframe/Iframe.jsx

```diff
@@ -19,8 +19,7 @@ function Switch({ organs_count }) {
       return (
         <ccf-organ-info
           organ-iri={iri}
-          use-remote-api="true"
-          remote-api-endpoint="https://apps.humanatlas.io/hubmap-hra-api/v1"
+          data-sources='["https://apps.humanatlas.io/api/ds-graph/hubmap?token="]'
```

Comment:
```suggestion
          data-sources='["https://apps.humanatlas.io/api/ds-graph/hubmap"]'
```
Do we need the empty token param?

Reply:
It is preferable to keep the `?token=` for caching purposes as the EUI uses the same data sources string and will be cached across both products.